Source/WSTagsField.swift

```diff
@@ -9,7 +9,14 @@
 import UIKit
 
 public protocol WSTagsFieldDelegate {
-    func tagInputFieldValidations() -> (valid: Bool, message: String?)
+    func tagInputFieldValidations() -> (valid: Bool, error: WSTagError?)
+}
+
+public protocol WSTagError {}
```

Comment:
isn't it better to rename this to WSTagValidationError or WSTagValidationReason ... to show it's a validation error and not a regular error?